apps/zbugs/src/components/avatar-image.tsx

```diff
@@ -1,9 +1,9 @@
+import type {Row} from '@rocicorp/zero';
 import {memo, type ImgHTMLAttributes} from 'react';
-import type {UserRow} from '../../shared/schema.ts';
 import {avatarURLWithSize} from '../avatar-url-with-size.ts';
 
 interface AvatarImageProps extends ImgHTMLAttributes<HTMLImageElement> {
-  user: UserRow;
+  user: Row['user'];
```

Comment:
Neat. Make sure this feature is covered in the commit message so it gets documented.